### PR TITLE
Blend policy intents across ensemble models

### PR DESCRIPTION
**Pull request by sann1432**
## Summary
- ensemble policy decisions across trend, mean-reversion, and volatility breakout intents with Sharpe-weighted voting and entropy-based abstention
- derive dynamic ATR-driven stop-loss/take-profit bands per symbol and expose optional model variant loading for the intent server
- update policy service tests to exercise the blended responses and new risk targets

## Testing
- pytest tests/test_policy_service_api.py
- pytest tests/policy/test_policy_service_v2.py
- pytest tests/policy/test_policy_service.py

------
https://chatgpt.com/codex/tasks/task_e_68dd65a7d2cc8321844d996f993ae593

### CODE REVIEW

File: policy_service.py [hunk 1]
```diff
@@ -182,74 +318,99 @@ async def decide_policy(request: PolicyDecisionRequest) -> PolicyDecisionRespons
...
         selected_template = max(action_templates, key=lambda template: template.edge_bps)
 
     fee_adjusted_edge = selected_template.edge_bps if selected_template else 0.0
+    approval_score = sum(
+        weights.get(variant, 0.0) for variant, intent in intents.items() if intent.approved
+    )
+
+    atr = _resolve_atr(request.instrument, book_snapshot, state_model)
+    take_profit = 3.0 * atr
+    stop_loss = 2.0 * atr
```
[the remark this answers is not in the document]
Comment (sann1432) [reply policy_service.py:386]: @codex fix comment
